tflite2onnx/op/squareddifference.py

```diff
@@ -0,0 +1,118 @@
+import logging
+import tflite
+import numpy as np
+
+from tflite2onnx import mapping
+from tflite2onnx.op.common import Operator
+from tflite2onnx.op.binary import Binary
+
+logger = logging.getLogger('tflite2onnx')
+
+
+# wrappers are used here to override Binary.type property
+# otherwise, Binary.type fails when trying to get self.tflite
+class SubtractWrapper(Binary):
+    @property
+    def type(self):
+        return 'Sub'
+
+
+class PowerWrapper(Binary):
+    @property
+    def type(self):
+        return 'Pow'
+
+
+class SquaredDifference(Operator):
+    # use identity to propagate difference output to square input
+    TypeMapping = {
+        tflite.BuiltinOperator.SQUARED_DIFFERENCE: 'Identity',
```

Comment:
What about making the _operator_ as a `Sub + Pow` directly such that we don't need the `Identity`?

Reply:
https://github.com/jackwish/tflite2onnx/pull/48/files#diff-1e482a50b4377b2620d1340cb86b9dbec61b7ccb8a7e7620955a16451624d943R1